### ISSUE TITLE
GRPC Node extension missing

### ISSUE THREAD

**Issue by tejasmanohar** [grpc/grpc#8373]
```
Error: Cannot find module '/Users/tejas/dev/src/github.com/segment-sources/stripe/node_modules/grpc/src/node/extension_binary/grpc_node.node'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/tejas/dev/src/github.com/segment-sources/stripe/node_modules/grpc/src/node/src/grpc_extension.js:38:15)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/tejas/dev/src/github.com/segment-sources/stripe/node_modules/grpc/src/node/src/client.js:55:12)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
ERRO[0005] exit status 1                                 X=github.com/segmentio/source-runner/source.go:380 (*Source).runApplication`
```

Hmmm, what could I be missing here?

```
$ brew upgrade grpc --with-plugins
Error: grpc/grpc/grpc 1.0.0 already installed
```

**Comment by murgatroid99** [comment]
I don't think the homebrew stuff is relevant. If you ran `npm install` to get the package, it should have downloaded or built the extension, unless you somehow prevented it from doing so.

It would help if you could run `npm install grpc` again in your `/Users/tejas/dev/src/github.com/segment-sources/stripe` and, if that doesn't fix the problem, provide the output of that command.

**Comment by tejasmanohar** [comment]
```
Tejass-MacBook-Pro:stripe tejas$ node -v
v6.3.1
Tejass-MacBook-Pro:stripe tejas$ npm i

> dtrace-provider@0.6.0 install /Users/tejas/dev/src/github.com/segment-sources/stripe/node_modules/dtrace-provider
> node scripts/install.js


> gc-stats@1.0.0 install /Users/tejas/dev/src/github.com/segment-sources/stripe/node_modules/gc-stats
> node-gyp rebuild

  CXX(target) Release/obj.target/gcstats/src/gcstats.o
  SOLINK_MODULE(target) Release/gcstats.node
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of OS X 10.9

> grpc@1.0.0 install /Users/tejas/dev/src/github.com/segment-sources/stripe/node_modules/grpc
> node-pre-gyp install --fallback-to-build

[grpc] Success: "/Users/tejas/dev/src/github.com/segment-sources/stripe/node_modules/grpc/src/node/extension_binary/grpc_node.node" is installed via remote
source-stripe@1.13.3 /Users/tejas/dev/src/github.com/segment-sources/stripe
├── @segment/eslint-config@3.4.0
├─┬ @segment/kit@3.1.1
│ ├─┬ bunyan@1.8.1
│ │ ├── dtrace-provider@0.6.0
│ │ ├── moment@2.15.1
│ │ ├─┬ mv@2.1.1
│ │ │ ├── ncp@2.0.0
│ │ │ └── rimraf@2.4.5
│ │ └── safe-json-stringify@1.0.3
│ ├── eventemitter2@1.0.5
│ ├─┬ gc-stats@1.0.0
│ │ ├── bindings@1.2.1
│ │ └── nan@2.4.0
│ ├── minimist@1.2.0
│ ├── node-statsd@0.1.1
│ ├── semver@5.3.0
│ ├── tcomb@2.7.0
│ ├── tcomb-validation@2.3.0
│ └── toobusy-js@0.4.3
├─┬ @segment/source@2.6.0
│ ├─┬ grpc@1.0.0
│ │ ├── arguejs@0.2.3
│ │ ├─┬ node-pre-gyp@0.6.29
│ │ │ ├─┬ mkdirp@0.5.1
│ │ │ │ └── minimist@0.0.8
│ │ │ ├─┬ nopt@3.0.6
│ │ │ │ └── abbrev@1.0.9
│ │ │ ├─┬ npmlog@3.1.2
│ │ │ │ ├─┬ are-we-there-yet@1.1.2
│ │ │ │ │ ├── delegates@1.0.0
│ │ │ │ │ └─┬ readable-stream@2.1.5
│ │ │ │ │   ├── buffer-shims@1.0.0
│ │ │ │ │   ├── core-util-is@1.0.2
│ │ │ │ │   ├── inherits@2.0.1
│ │ │ │ │   ├── isarray@1.0.0
│ │ │ │ │   ├── process-nextick-args@1.0.7
│ │ │ │ │   ├── string_decoder@0.10.31
│ │ │ │ │   └── util-deprecate@1.0.2
│ │ │ │ ├── console-control-strings@1.1.0
│ │ │ │ ├─┬ gauge@2.6.0
│ │ │ │ │ ├── aproba@1.0.4
│ │ │ │ │ ├── has-color@0.1.7
│ │ │ │ │ ├── has-unicode@2.0.1
│ │ │ │ │ ├── object-assign@4.1.0
│ │ │ │ │ ├── signal-exit@3.0.0
│ │ │ │ │ ├─┬ string-width@1.0.2
│ │ │ │ │ │ ├─┬ code-point-at@1.0.0
│ │ │ │ │ │ │ └── number-is-nan@1.0.0
│ │ │ │ │ │ └─┬ is-fullwidth-code-point@1.0.0
│ │ │ │ │ │   └── number-is-nan@1.0.0
│ │ │ │ │ ├─┬ strip-ansi@3.0.1
│ │ │ │ │ │ └── ansi-regex@2.0.0
│ │ │ │ │ └── wide-align@1.1.0
│ │ │ │ └── set-blocking@2.0.0
│ │ │ ├─┬ rc@1.1.6
│ │ │ │ ├── deep-extend@0.4.1
│ │ │ │ ├── ini@1.3.4
│ │ │ │ ├── minimist@1.2.0
│ │ │ │ └── strip-json-comments@1.0.4
│ │ │ ├─┬ request@2.74.0
│ │ │ │ ├── aws-sign2@0.6.0
│ │ │ │ ├── aws4@1.4.1
│ │ │ │ ├─┬ bl@1.1.2
│ │ │ │ │ └─┬ readable-stream@2.0.6
│ │ │ │ │   ├── core-util-is@1.0.2
│ │ │ │ │   ├── inherits@2.0.1
│ │ │ │ │   ├── isarray@1.0.0
│ │ │ │ │   ├── process-nextick-args@1.0.7
│ │ │ │ │   ├── string_decoder@0.10.31
│ │ │ │ │   └── util-deprecate@1.0.2
│ │ │ │ ├── caseless@0.11.0
│ │ │ │ ├─┬ combined-stream@1.0.5
│ │ │ │ │ └── delayed-stream@1.0.0
│ │ │ │ ├── extend@3.0.0
│ │ │ │ ├── forever-agent@0.6.1
│ │ │ │ ├─┬ form-data@1.0.0-rc4
│ │ │ │ │ └── async@1.5.2
│ │ │ │ ├─┬ har-validator@2.0.6
│ │ │ │ │ ├─┬ chalk@1.1.3
│ │ │ │ │ │ ├── ansi-styles@2.2.1
│ │ │ │ │ │ ├── escape-string-regexp@1.0.5
│ │ │ │ │ │ ├─┬ has-ansi@2.0.0
│ │ │ │ │ │ │ └── ansi-regex@2.0.0
│ │ │ │ │ │ ├─┬ strip-ansi@3.0.1
│ │ │ │ │ │ │ └── ansi-regex@2.0.0
│ │ │ │ │ │ └── supports-color@2.0.0
│ │ │ │ │ ├─┬ commander@2.9.0
│ │ │ │ │ │ └── graceful-readlink@1.0.1
│ │ │ │ │ ├─┬ is-my-json-valid@2.13.1
│ │ │ │ │ │ ├── generate-function@2.0.0
│ │ │ │ │ │ ├─┬ generate-object-property@1.2.0
│ │ │ │ │ │ │ └── is-property@1.0.2
│ │ │ │ │ │ ├── jsonpointer@2.0.0
│ │ │ │ │ │ └── xtend@4.0.1
│ │ │ │ │ └─┬ pinkie-promise@2.0.1
│ │ │ │ │   └── pinkie@2.0.4
│ │ │ │ ├─┬ hawk@3.1.3
│ │ │ │ │ ├── boom@2.10.1
│ │ │ │ │ ├── cryptiles@2.0.5
│ │ │ │ │ ├── hoek@2.16.3
│ │ │ │ │ └── sntp@1.0.9
│ │ │ │ ├─┬ http-signature@1.1.1
│ │ │ │ │ ├── assert-plus@0.2.0
│ │ │ │ │ ├─┬ jsprim@1.3.0
│ │ │ │ │ │ ├── extsprintf@1.0.2
│ │ │ │ │ │ ├── json-schema@0.2.2
│ │ │ │ │ │ └── verror@1.3.6
│ │ │ │ │ └─┬ sshpk@1.9.2
│ │ │ │ │   ├── asn1@0.2.3
│ │ │ │ │   ├── assert-plus@1.0.0
│ │ │ │ │   ├── dashdash@1.14.0
│ │ │ │ │   ├── ecc-jsbn@0.1.1
│ │ │ │ │   ├── getpass@0.1.6
│ │ │ │ │   ├── jodid25519@1.0.2
│ │ │ │ │   ├── jsbn@0.1.0
│ │ │ │ │   └── tweetnacl@0.13.3
│ │ │ │ ├── is-typedarray@1.0.0
│ │ │ │ ├── isstream@0.1.2
│ │ │ │ ├── json-stringify-safe@5.0.1
│ │ │ │ ├─┬ mime-types@2.1.11
│ │ │ │ │ └── mime-db@1.23.0
│ │ │ │ ├── node-uuid@1.4.7
│ │ │ │ ├── oauth-sign@0.8.2
│ │ │ │ ├── qs@6.2.1
│ │ │ │ ├── stringstream@0.0.5
│ │ │ │ ├── tough-cookie@2.3.1
│ │ │ │ └── tunnel-agent@0.4.3
│ │ │ ├─┬ rimraf@2.5.4
│ │ │ │ └─┬ glob@7.0.5
│ │ │ │   ├── fs.realpath@1.0.0
│ │ │ │   ├─┬ inflight@1.0.5
│ │ │ │   │ └── wrappy@1.0.2
│ │ │ │   ├── inherits@2.0.1
│ │ │ │   ├─┬ minimatch@3.0.3
│ │ │ │   │ └─┬ brace-expansion@1.1.6
│ │ │ │   │   ├── balanced-match@0.4.2
│ │ │ │   │   └── concat-map@0.0.1
│ │ │ │   ├─┬ once@1.3.3
│ │ │ │   │ └── wrappy@1.0.2
│ │ │ │   └── path-is-absolute@1.0.0
│ │ │ ├── semver@5.2.0
│ │ │ ├─┬ tar@2.2.1
│ │ │ │ ├── block-stream@0.0.9
│ │ │ │ ├─┬ fstream@1.0.10
│ │ │ │ │ └── graceful-fs@4.1.5
│ │ │ │ └── inherits@2.0.1
│ │ │ └─┬ tar-pack@3.1.4
│ │ │   ├─┬ debug@2.2.0
│ │ │   │ └── ms@0.7.1
│ │ │   ├─┬ fstream@1.0.10
│ │ │   │ ├── graceful-fs@4.1.5
│ │ │   │ └── inherits@2.0.1
│ │ │   ├─┬ fstream-ignore@1.0.5
│ │ │   │ ├── inherits@2.0.1
│ │ │   │ └─┬ minimatch@3.0.3
│ │ │   │   └─┬ brace-expansion@1.1.6
│ │ │   │     ├── balanced-match@0.4.2
│ │ │   │     └── concat-map@0.0.1
│ │ │   ├─┬ once@1.3.3
│ │ │   │ └── wrappy@1.0.2
│ │ │   ├─┬ readable-stream@2.1.5
│ │ │   │ ├── buffer-shims@1.0.0
│ │ │   │ ├── core-util-is@1.0.2
│ │ │   │ ├── inherits@2.0.1
│ │ │   │ ├── isarray@1.0.0
│ │ │   │ ├── process-nextick-args@1.0.7
│ │ │   │ ├── string_decoder@0.10.31
│ │ │   │ └── util-deprecate@1.0.2
│ │ │   └── uid-number@0.0.6
│ │ └─┬ protobufjs@4.1.3
│ │   ├─┬ ascli@1.0.0
│ │   │ ├── colour@0.7.1
│ │   │ └── optjs@3.2.2
│ │   ├─┬ bytebuffer@4.1.0
│ │   │ ├── bufferview@1.0.1
│ │   │ └── long@2.4.0
│ │   ├── glob@5.0.15
│ │   └─┬ yargs@3.32.0
│ │     ├── camelcase@2.1.1
│ │     ├─┬ cliui@3.2.0
│ │     │ └── wrap-ansi@2.0.0
│ │     ├── decamelize@1.2.0
│ │     ├─┬ os-locale@1.4.0
│ │     │ └─┬ lcid@1.0.0
│ │     │   └── invert-kv@1.0.0
│ │     ├── window-size@0.1.4
│ │     └── y18n@3.2.1
│ ├─┬ nodeify@1.0.0
│ │ ├── is-promise@1.0.1
│ │ └── promise@1.3.0
│ └── pify@2.3.0
├── bluebird@3.4.6
├─┬ change-case@2.3.1
│ ├── camel-case@1.2.2
│ ├── constant-case@1.1.2
│ ├── dot-case@1.1.2
│ ├── is-lower-case@1.1.3
│ ├── is-upper-case@1.1.2
│ ├── lower-case@1.1.3
│ ├── lower-case-first@1.0.2
│ ├── param-case@1.1.2
│ ├── pascal-case@1.1.2
│ ├── path-case@1.1.2
│ ├── sentence-case@1.1.3
│ ├── snake-case@1.1.2
│ ├── swap-case@1.1.2
│ ├── title-case@1.1.2
│ ├── upper-case@1.1.3
│ └── upper-case-first@1.1.2
├─┬ co-request@1.0.0
│ └─┬ request@2.75.0
│   ├── aws-sign2@0.6.0
│   ├── aws4@1.5.0
│   ├── bl@1.1.2
│   ├── caseless@0.11.0
│   ├─┬ combined-stream@1.0.5
│   │ └── delayed-stream@1.0.0
│   ├── forever-agent@0.6.1
│   ├─┬ form-data@2.0.0
│   │ └── asynckit@0.4.0
│   ├─┬ har-validator@2.0.6
│   │ ├─┬ commander@2.9.0
│   │ │ └── graceful-readlink@1.0.1
│   │ └─┬ pinkie-promise@2.0.1
│   │   └── pinkie@2.0.4
│   ├─┬ hawk@3.1.3
│   │ ├── boom@2.10.1
│   │ ├── cryptiles@2.0.5
│   │ ├── hoek@2.16.3
│   │ └── sntp@1.0.9
│   ├─┬ http-signature@1.1.1
│   │ ├── assert-plus@0.2.0
│   │ ├─┬ jsprim@1.3.1
│   │ │ ├── extsprintf@1.0.2
│   │ │ ├── json-schema@0.2.3
│   │ │ └── verror@1.3.6
│   │ └─┬ sshpk@1.10.1
│   │   ├── asn1@0.2.3
│   │   ├── assert-plus@1.0.0
│   │   ├── bcrypt-pbkdf@1.0.0
│   │   ├─┬ dashdash@1.14.0
│   │   │ └── assert-plus@1.0.0
│   │   ├── ecc-jsbn@0.1.1
│   │   ├─┬ getpass@0.1.6
│   │   │ └── assert-plus@1.0.0
│   │   ├── jodid25519@1.0.2
│   │   ├── jsbn@0.1.0
│   │   └── tweetnacl@0.14.3
│   ├── is-typedarray@1.0.0
│   ├── isstream@0.1.2
│   ├── json-stringify-safe@5.0.1
│   ├─┬ mime-types@2.1.12
│   │ └── mime-db@1.24.0
│   ├── node-uuid@1.4.7
│   ├── oauth-sign@0.8.2
│   ├── qs@6.2.1
│   ├── stringstream@0.0.5
│   ├── tough-cookie@2.3.1
│   └── tunnel-agent@0.4.3
├─┬ defaults@1.0.3
│ └── clone@1.0.2
├─┬ dependency-check@2.6.0
│ ├── async@1.5.2
│ ├── builtins@1.0.3
│ ├─┬ debug@2.2.0
│ │ └── ms@0.7.1
│ ├─┬ detective@4.3.1
│ │ ├── acorn@1.2.2
│ │ └── defined@1.0.0
│ ├─┬ is-relative@0.2.1
│ │ └─┬ is-unc-path@0.1.1
│ │   └── unc-path-regex@0.1.2
│ ├─┬ read-package-json@2.0.4
│ │ ├── glob@6.0.4
│ │ ├── graceful-fs@4.1.9
│ │ ├─┬ json-parse-helpfulerror@1.0.3
│ │ │ └── jju@1.3.0
│ │ └─┬ normalize-package-data@2.3.5
│ │   ├── hosted-git-info@2.1.5
│ │   ├── is-builtin-module@1.0.0
│ │   └─┬ validate-npm-package-license@3.0.1
│ │     ├─┬ spdx-correct@1.0.2
│ │     │ └── spdx-license-ids@1.2.2
│ │     └── spdx-expression-parse@1.0.4
│ └── resolve@1.1.7
├─┬ eslint@2.13.1
│ ├─┬ chalk@1.1.3
│ │ ├── ansi-styles@2.2.1
│ │ ├── escape-string-regexp@1.0.5
│ │ ├── has-ansi@2.0.0
│ │ ├── strip-ansi@3.0.1
│ │ └── supports-color@2.0.0
│ ├─┬ concat-stream@1.5.2
│ │ ├── inherits@2.0.3
│ │ ├─┬ readable-stream@2.0.6
│ │ │ ├── core-util-is@1.0.2
│ │ │ ├── process-nextick-args@1.0.7
│ │ │ └── string_decoder@0.10.31
│ │ └── typedarray@0.0.6
│ ├─┬ doctrine@1.5.0
│ │ └── isarray@1.0.0
│ ├─┬ es6-map@0.1.4
│ │ ├── d@0.1.1
│ │ ├── es5-ext@0.10.12
│ │ ├── es6-iterator@2.0.0
│ │ ├── es6-set@0.1.4
│ │ ├── es6-symbol@3.1.0
│ │ └── event-emitter@0.3.4
│ ├─┬ escope@3.6.0
│ │ ├── es6-weak-map@2.0.1
│ │ └─┬ esrecurse@4.1.0
│ │   └── estraverse@4.1.1
│ ├─┬ espree@3.3.2
│ │ ├── acorn@4.0.3
│ │ └─┬ acorn-jsx@3.0.1
│ │   └── acorn@3.3.0
│ ├── estraverse@4.2.0
│ ├── esutils@2.0.2
│ ├─┬ file-entry-cache@1.3.1
│ │ ├─┬ flat-cache@1.2.1
│ │ │ ├── circular-json@0.3.1
│ │ │ ├─┬ del@2.2.2
│ │ │ │ ├─┬ globby@5.0.0
│ │ │ │ │ ├─┬ array-union@1.0.2
│ │ │ │ │ │ └── array-uniq@1.0.3
│ │ │ │ │ ├── arrify@1.0.1
│ │ │ │ │ └── glob@7.1.1
│ │ │ │ ├── is-path-cwd@1.0.0
│ │ │ │ └─┬ is-path-in-cwd@1.0.0
│ │ │ │   └── is-path-inside@1.0.0
│ │ │ └── write@0.2.1
│ │ └── object-assign@4.1.0
│ ├─┬ glob@7.1.1
│ │ ├── fs.realpath@1.0.0
│ │ ├─┬ inflight@1.0.6
│ │ │ └── wrappy@1.0.2
│ │ ├─┬ minimatch@3.0.3
│ │ │ └─┬ brace-expansion@1.1.6
│ │ │   ├── balanced-match@0.4.2
│ │ │   └── concat-map@0.0.1
│ │ └── once@1.4.0
│ ├── globals@9.12.0
│ ├── ignore@3.2.0
│ ├── imurmurhash@0.1.4
│ ├─┬ inquirer@0.12.0
│ │ ├── ansi-escapes@1.4.0
│ │ ├── ansi-regex@2.0.0
│ │ ├─┬ cli-cursor@1.0.2
│ │ │ └─┬ restore-cursor@1.0.1
│ │ │   ├── exit-hook@1.1.1
│ │ │   └── onetime@1.1.0
│ │ ├── cli-width@2.1.0
│ │ ├── figures@1.7.0
│ │ ├── lodash@4.16.4
│ │ ├─┬ readline2@1.0.1
│ │ │ ├─┬ code-point-at@1.0.1
│ │ │ │ └── number-is-nan@1.0.1
│ │ │ ├── is-fullwidth-code-point@1.0.0
│ │ │ └── mute-stream@0.0.5
│ │ ├── run-async@0.1.0
│ │ ├── rx-lite@3.1.2
│ │ ├── string-width@1.0.2
│ │ └── through@2.3.8
│ ├─┬ is-my-json-valid@2.15.0
│ │ ├── generate-function@2.0.0
│ │ ├─┬ generate-object-property@1.2.0
│ │ │ └── is-property@1.0.2
│ │ ├── jsonpointer@4.0.0
│ │ └── xtend@4.0.1
│ ├─┬ is-resolvable@1.0.0
│ │ └── tryit@1.0.2
│ ├─┬ js-yaml@3.6.1
│ │ ├── argparse@1.0.9
│ │ └── esprima@2.7.3
│ ├─┬ json-stable-stringify@1.0.1
│ │ └── jsonify@0.0.0
│ ├─┬ levn@0.3.0
│ │ ├── prelude-ls@1.1.2
│ │ └── type-check@0.3.2
│ ├── lodash@4.16.4
│ ├─┬ mkdirp@0.5.1
│ │ └── minimist@0.0.8
│ ├─┬ optionator@0.8.2
│ │ ├── deep-is@0.1.3
│ │ ├── fast-levenshtein@2.0.5
│ │ └── wordwrap@1.0.0
│ ├── path-is-absolute@1.0.1
│ ├── path-is-inside@1.0.2
│ ├── pluralize@1.2.1
│ ├── progress@1.1.8
│ ├─┬ require-uncached@1.0.2
│ │ ├─┬ caller-path@0.1.0
│ │ │ └── callsites@0.2.0
│ │ └── resolve-from@1.0.1
│ ├── shelljs@0.6.1
│ ├── strip-json-comments@1.0.4
│ ├─┬ table@3.8.0
│ │ ├─┬ ajv@4.7.7
│ │ │ └── co@4.6.0
│ │ ├── ajv-keywords@1.1.1
│ │ ├── lodash@4.16.4
│ │ └── slice-ansi@0.0.4
│ ├── text-table@0.2.0
│ └─┬ user-home@2.0.0
│   └── os-homedir@1.0.2
├── UNMET PEER DEPENDENCY eslint-plugin-mocha@^2.0.0
├── UNMET PEER DEPENDENCY eslint-plugin-react@^4.0.0
├─┬ eslint-plugin-require-path-exists@1.1.5
│ ├── builtin-modules@1.1.1
│ └─┬ fs-plus@2.9.3
│   ├── async@0.2.10
│   ├── mkdirp@0.3.5
│   ├── rimraf@2.2.8
│   └─┬ underscore-plus@1.6.6
│     └── underscore@1.6.0
├── extend@3.0.0
├─┬ flat@1.6.1
│ └── is-buffer@1.1.4
├── lodash@3.10.1
├─┬ md5@2.2.1
│ ├── charenc@0.0.1
│ └── crypt@0.0.1
├─┬ token-bucket-promise@1.0.1
│ ├─┬ babel-polyfill@6.16.0
│ │ ├── babel-runtime@6.11.6
│ │ ├── core-js@2.4.1
│ │ └── regenerator-runtime@0.9.5
│ └── delay@1.3.1
└─┬ underscore.string@3.3.4
  ├── sprintf-js@1.0.3
  └── util-deprecate@1.0.2

npm WARN @segment/eslint-config@3.4.0 requires a peer of eslint-plugin-mocha@^2.0.0 but none was installed.
npm WARN @segment/eslint-config@3.4.0 requires a peer of eslint-plugin-react@^4.0.0 but none was installed.
$ npm -v
3.10.3
```

**Comment by murgatroid99** [comment]
So, that output has the line `[grpc] Success: "/Users/tejas/dev/src/github.com/segment-sources/stripe/node_modules/grpc/src/node/extension_binary/grpc_node.node" is installed via remote`. Are you still getting a message that that file can't be found? Can you confirm that the file actually exists in the file system?

**Comment by tejasmanohar** [comment]
@murgatroid99 Required an `npm rebuild`. Not sure why

**Comment by itsmikeferrari** [comment]
"npm rebuild" worked for me as well. I was experiencing this type of error while deploying to Firebase from the command line. It suggested I run npm install in my functions directory, and I did but that didn't solve the problem.

After installing every package npm was WARN ing about, I used "npm rebuild", then when I ran "firebase deploy" again it worked.

I wish I would have know about "npm rebuild" before installing a bunch of packages that had WARN next to the output in the terminal. I'm still a bit new to npm and my use of the force is not as strong as others. yet.
